README.md

```diff
@@ -123,7 +124,7 @@ $ docker run -p 8080:80 -e SERVER_NAME=myhost my-modsec
 | METRICS_ALLOW_FROM  | A string indicating a range of IP adresses that can access the metrics (Default: `127.0.0.0/255.0.0.0 ::1/128`) | 
 | METRICS_DENY_FROM  | A string indicating a range of IP adresses that cannot access the metrics (Default: `All`) | 
 | METRICSLOG  | A string indicating the path of the metrics log (Default: `/dev/null combined`) | 
-| PERFLOG  | A string indicating the path of the performance log (Default: `/dev/stdout  perflogjson env=write_perflog`) |
+| PERFLOG  | A string indicating the path of the performance log (Default: `/dev/stdout perflogjson env=write_perflog`) |
```

Comment:
I couldn't find any `LogFormat` definition for `perflogjson`. How can this work? Maybe it works because `write_perfolg` is never set? From my point of view we need
1. a `LogFormat` definition of `perflogjson`
2.  a default for `write_perflog` (or `WRITE_PERFLOG`, preferrably)

Reply:
Probably that was lost in time. We need to find out the right format in git history probably

Reply:
That definition has never existed. The name was introduced in cfe220bf2cfdf3d0d85a912ca7cecf5ef8742e8f, where most of the configuration variables were first introduced.

https://github.com/coreruleset/modsecurity-docker/issues/128

Reply:
Excellent, let's move this to the new issue.